ledger-tool/src/main.rs

```diff
@@ -1023,7 +1039,13 @@ fn main() {
                 ..ProcessOptions::default()
             };
             let genesis_config = open_genesis_config_by(&ledger_path, arg_matches);
-            match load_bank_forks(arg_matches, &ledger_path, &genesis_config, process_options) {
+            match load_bank_forks(
```

Comment:
TODO: Very sadly, `create-snapshot` is a bit broken for different reason.... I need yet another pr, first.....

Reply:
TODO: triage it and create an issue or pr.

Reply:
done #10212 (ci pending)